docs/generator.py

```diff
@@ -37,8 +40,11 @@ def __init__(self, template_dir, template_ctx):
                           extensions=['docs.extension.IncludePlaybookTasks'])
         env.filters['camel_to_snake'] = camel_to_snake
         env.filters['split_operation_names'] = split_operation_names
+        env.filters['show_description_with_references'] = show_description_with_references
+        env.filters['get_link_to_model_page_by_name'] = get_link_to_model_page_by_name
```

Comment:
`BaseDocGenerator` does not know about models and references, so I'd rather move these filters to `ApiSpecDocGenerator` to narrow their visibility and limit the scope. Anyway, only api-related templates use these filters.